docker-compose.yaml

```diff
@@ -0,0 +1,210 @@
+services:
+  mssql:
+    build:
+      context: ./containers/db
+      args:
+        - DATABASE_PASSWORD=${DATABASE_PASSWORD:-fake.fake.fake.1234}
+    container_name: rtr-mssql
```

Comment:
would it be helpful to define the platform for the mssql service? I'm getting the warning `The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested` when I bring the container up, although it still starts